### DISCUSSION TITLE
About WorkFlow component

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#48843] [Q&A]
Hi. The topic of WorkFlow is a bit complicated for me and I did not understand when to use it. Can someone explain this to me with a simple example?
https://symfony.com/doc/current/workflow.html

**Comment by 94noni** [comment]
a workflow is great to have on an object, usually entities but not mandatory, to controle its state
the example https://symfony.com/doc/current/workflow/workflow-and-state-machine.html#example showcase a pull request with its state, and controle by the workflow

if this example is not clear enough, can you just describe what you dont understand in this simple example ?
so we can improve the documentation :)
